src/test/run-make/symbol-visibility/Makefile

```diff
@@ -0,0 +1,26 @@
+include ../tools.mk
+
+all:
```

Comment:
This'll probably need a guard or two for only running on Linux (I think). 

Either that or it'll need some tweaks to run on OSX/Windows